Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7534.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7534, "Span with tail truncation and paragraph breaks with Java.Lang.IndexOutOfBoundsException", PlatformAffected.Android)]
+	public partial class Issue7534 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var span = new Span
+			{
+				Text =
+				" Mi augue molestie ligula lobortis enim Velit, in. \n Imperdiet eu dignissim odio. Massa erat Hac inceptos facilisis nibh " +
+				" Interdum massa Consectetuer risus sociis molestie facilisi enim. Class gravida. \n Gravida sociosqu cras Quam velit, suspendisse" +
+				"  leo auctor odio integer primis dui potenti dolor faucibus augue justo morbi ornare sem. "
+			};
+
+			var formattedString = new FormattedString();
+			formattedString.Spans.Add(span);
+
+			var label = new Xamarin.Forms.Label
+			{
+				LineBreakMode = LineBreakMode.TailTruncation,
+				FormattedText = formattedString,
+				MaxLines = 3 
+				//max line is less than the text reproduce and textViewExtensions couldn't identify when
+			};
+
+			var layout = new Xamarin.Forms.StackLayout();
+			layout.Children.Add(label);
```

Comment:
Could you maybe add a simple label describing what is happening. Some thing like: "If you opened this page, the app didn't crash and you can read three lines in the label below, this test has passed"

Reply:
I think I gonna put exactly what you write. :) Can I?

Reply:
If that is an accurate description of how this test works, definitely!

Reply:
Ah, I'm sorry I wasn't completely clear. I also meant add it as a Label to the actual screen, not just as a comment in code

Reply:
Yeah, I saw that this is more an "add" to the UITest you suggested I should do right? Anyway, I'm gonna do it along with the UITest change commit I'll do later. But along with the label I should put on screen, I can keep the long one too, right? Because I think a long label should be needed to test more large screen variations.